src/index.html

```diff
@@ -1,14 +1,50 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
+    <title>Moyo header</title>
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home" class="logo__img">
+        <img class="logo__image" src="./images/logo.png"
+        alt="Moyo logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__item">
+          <li class="nav__list">
+            <a class="nav__link  is-active" href="#home">Apple</a>
+          </li>
+          <li class="nav__list">
+            <a class="nav__link" href="#home">Samsung</a>
+          </li>
+          <li class="nav__list">
+            <a class="nav__link" href="#home">Smartphones</a>
+          </li>
+          <li class="nav__list" >
+            <a class="nav__link" data-qa="hover"
+             href="#home">Laptops & Computers</a>
```

Comment:
fix the code formatting
```suggestion
            <a
              class="nav__link"
              data-qa="hover"
              href="#home"
            >
              Laptops & Computers
            </a>
```